ckanext/validation/jobs.py

```diff
@@ -286,14 +292,14 @@ def _read_excel_file(data, extension=None):
     return df
 
 
-def _read_json_file(json_path):
+def _read_json_file(json_url):
     """
     Reads a json or geojson file in as a pandas dataframe ready for validation.
     """
     # Load as plain JSON
     try:
-        with open(json_path, 'r') as read_file:
-            geojson = json.load(read_file)
+        r = requests.get(json_url)
+        geojson = json.load(StringIO(r.content))
```

Comment:
Not too sure why we need the `StringIO()`,
but this should do the trick if it's just a json response:

```suggestion
        geojson = r.json()
```

Reply:
Good point. I've included this in the PR.